// localized-readability/dist/annotations/language.en-us.js
(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.Annotations = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
module.exports = {
  general: {
    module: "Readability",
    score: "Score",
    interpretations: "Interpretations",
    counts: "Counts",
    age: "Age",
    grade: "Grade",
    paragraphs: "Paragraphs",
    sentences: "Sentences",
    words: "Words",
    syllables: "Syllables",
    polysillabicWords: "Polysillabic words",
    letters: "Letters",
    characters: "Characters",
    longwords: "Long words",
    periods: "Periods",
    punctuations: "Punctuations",
    whitespaces: "Whitespaces"
  },
  modifier: {
    grade: {
      constant: 5,
      source: "https://en.wikipedia.org/wiki/Educational_stage#United_States"
    },
    syllable: {
      constant: 1,
      source: ""
    }
  },
  highlight: {
    "0": "Normal",
    "1": "Fairly difficult",
    "2": "Difficult",
    "3": "Very difficult",
    "4": "Extremely difficult"
  },
  stats: {
    consensus: {
      description: "Average across measurements."
    },
    automatedReadability: {
      title: "Automated Readability Index",
      description:
        "Approximates US grade level, using the amount of sentences, words, and letters."
    },
    flesch: {
      title: "Flesch Reading Ease",
      description:
        "Measures complexity of text, using the average length of sentences, and syllables in words.",
      levels: {
        "0": "Very difficult",
        "1": "Difficult",
        "2": "Fairly difficult",
        "3": "Plain English.",
        "4": "Fairly easy",
        "5": "Easy",
        "6": "Very easy"
      },
      descriptor: "to read"
    },
    fleschKincaid: {
      title: "Flesch-Kincaid Grade-level",
      description: "Translates Flesch-score into US grade level."
    },
    colemanLiau: {
      title: "Coleman-Liau Grade-level",
      description:
        "Approximates US grade level, using the average number of sentences, words, and letters."
    },
    smog: {
      title: "Simple Measure of Gobbledygook",
      description:
        "Measures the amount of years of education needed to understand the text, using the amount of sentences and polysyllabic words."
    },
    lix: {
      title: "Lesbarhetsindex",
      description:
        "Reading ease based on total words, number of sentences and number of long words (more than 6 characters).",
      levels: {
        "0": "Very difficult",
        "1": "Difficult",
        "2": "Average",
        "3": "Easy",
        "4": "Very easy"
      }
    },
    rix: {
      title: "Anderson's Readability Index",
      description:
        "Translates LIX into grade level, using the amount of sentences and long words."
    }
  }
};

},{}]},{},[1])(1)
});
